docs/reference/tab-widgets/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -210,6 +210,21 @@ POST _snapshot/my_repository/snapshot-20200617/_restore
 <1> The indices to restore.
 +
 <2> We also want to restore the aliases.
++
+NOTE: If any <<feature-state,feature states>> need to be restored we'll need to specify them using the
+`feature_states` field and the indices that belong to the feature states we restore must not be specified under `indices`.
+The <<health-api, Health API>> returns both the `indices` and `feature_states` that need to be restored for the restore from snapshot diagnosis. e.g.:
++
```

Comment:
It's rather unfortunate we can't recommend something better here because there currently is no way of getting the system indices/data streams associated with a feature state ( https://github.com/elastic/elasticsearch/issues/86307 )